src/theme/variables.scss

```diff
@@ -74,6 +74,77 @@
   --ion-color-light-contrast-rgb: 0, 0, 0;
   --ion-color-light-shade: #d7d8da;
   --ion-color-light-tint: #f5f6f9;
+
```

Comment:
I would really like to see the theme affect the background colors and the rest of the app